SpatialGDK/Source/SpatialGDK/Private/EngineClasses/RPCContainer.h

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Schema/RPCPayload.h"
+#include "SpatialConstants.h"
+
+#include "CoreMinimal.h"
+
+// TODO(Alex): remove this logic when singletons can be referenced without entity IDs (UNR-1456) - deprecated???
+struct FPendingRPCParams
+{
+	FPendingRPCParams(UObject* InTargetObject, UFunction* InFunction, SpatialGDK::RPCPayload&& InPayload, int InReliableRPCIndex = 0);
+	~FPendingRPCParams() = default;
```

Comment:
We don't need to define a destructor here. Rule of 1.

Reply:
Removed.